randovania/games/am2r/json_data/Main Caves.txt

```diff
@@ -1243,7 +1247,12 @@ Extra - minimap_data: [{'x': 51, 'y': 29}, {'x': 51, 'y': 30}, {'x': 51, 'y': 31
           Space Jump Wall
           Any of the following:
               Space Jump or Can Use Spider Ball
-              Enabled Septogg Helpers and Hijump Wall
+              All of the following:
+                  Enabled Septogg Helpers
+                  Any of the following:
+                      Hijump Wall
```

Comment:
Maybe comment htis as "reach upper part with septogg" and "reach upper septogg" or something